### PR TITLE
scripted-diff: gitian: Use REFERENCE_DATETIME directly.

### PR DESCRIPTION
**Pull request by dongcarl**
Fixes regression introduced by #16141.

```
-BEGIN VERIFY SCRIPT-
sed -i 's#\$REFERENCE_DATE\\\\\\ \$REFERENCE_TIME#\$REFERENCE_DATETIME#g' contrib/gitian-descriptors/*
-END VERIFY SCRIPT-
```

-----

Note that this could have been fixed by escaping properly, but using `REFERENCE_DATETIME` directly is simpler.

Future note: `REFERENCE_{DATE{,DATETIME},TIME}` is a bit ridiculous. At the very _least_ gitian should use epoch, as it is the most parse-able, and preferably set SOURCE_DATE_EPOCH.

### CODE REVIEW

**Comment by MarcoFalke** [comment]
Can you explain how the regression was introduced?

**Comment by MarcoFalke** [comment]
Concept ACK

**Comment by fanquake** [comment]
Concept ACK. Thanks.

@MarcoFalke It was my poor copy-paste, followed by some over-the-shoulder review.

Also a little related discussion in the `#bitcoin-builds` IRC channel.

**Comment by MarcoFalke** [comment]
Oh, let me rephrase: Why did `REFERENCE_TIME` && `REFERENCE_DATE` work previously, but does no longer?

**Comment by dongcarl** [comment]
> Oh, let me rephrase: Why did `REFERENCE_TIME` && `REFERENCE_DATE` work previously, but does no longer?

@MarcoFalke 
https://gist.github.com/dongcarl/c1032fd0236efd62ebc369931c339577

**Comment by MarcoFalke** [comment]
Ah, so the issue could be fixed by escaping properly, but you decided to use `_DATETIME` directly. Sounds fair. Maybe mention that in the OP?

**Comment by dongcarl** [comment]
> Ah, so the issue could be fixed by escaping properly, but you decided to use `_DATETIME` directly. Sounds fair. Maybe mention that in the OP?

Done.

**Comment by sidhujag** [comment]
> Oh, let me rephrase: Why did `REFERENCE_TIME` && `REFERENCE_DATE` work previously, but does no longer?

 I don't think it was actually being used, it was in a variable that was removed and was put in by #16141 which broke the build

**Comment by dongcarl** [comment]
> I don't think it was actually being used, it was in a variable and then copied and pasted into the sed by #16141 which broke the build, but should have been escaped

@sidhujag It was being used pre-16141, see: https://www.gnu.org/software/tar/manual/html_node/using-tar-options.html

**Comment by dongcarl** [comment]
```
-----BEGIN PGP SIGNED MESSAGE-----
Hash: SHA256

ab4be93795002308de86cd02a699dd82e53bf4794b11095e651a57e7839d927a  bitcoin-0.18.99-aarch64-linux-gnu-debug.tar.gz
b9f3879221d42e6eb836a9de8641aee11359a7bf361020d668f3c5d511f37992  bitcoin-0.18.99-aarch64-linux-gnu.tar.gz
e627d30341d80eed6b4d7b74ff1b377e6cdccbe7c39173a0b21fc7bee89ac2f9  bitcoin-0.18.99-arm-linux-gnueabihf-debug.tar.gz
c9c0c1c798244770a0212dbd1dbfe53061f34539264933284ee027132e98d600  bitcoin-0.18.99-arm-linux-gnueabihf.tar.gz
abf7597ffe132a37f837bc5a239f0d7b2d712a9a85f5a752a5cace2181648338  bitcoin-0.18.99-i686-pc-linux-gnu-debug.tar.gz
76bedf0ad53e5e699f8bcc6ae00ccde20a927809a10d67682ddf61bb196fcf59  bitcoin-0.18.99-i686-pc-linux-gnu.tar.gz
3a1d541061568bee5282f918ef21e6ede14c9d8e4676110fa82bfda6608da729  bitcoin-0.18.99-riscv64-linux-gnu-debug.tar.gz
87e2fd3e11dbe5b9dfecb4a0ddb7e966af561be3f035360af2a8aefec010b746  bitcoin-0.18.99-riscv64-linux-gnu.tar.gz
4372290bfefa8b65904ba0a2df6efc3ad26f476da7f1c5367f7efee2592ea33d  bitcoin-0.18.99-x86_64-linux-gnu-debug.tar.gz
276f7039c7f0e8038373cf2a8ab9a52d237f076cfb243921f3d3b99ae870d48c  bitcoin-0.18.99-x86_64-linux-gnu.tar.gz
a09c58d9de205de9cb5f56ee7d02c524e275ed6d7e6aaecfb47c3718ac15295e  src/bitcoin-0.18.99.tar.gz
f6e72f3efcd2b97410f8fb9428c25e1fcdbfcae83dedb73a45a3e8ce0e087807  bitcoin-core-linux-0.19-res.yml
-----BEGIN PGP SIGNATURE-----

iQIzBAEBCAAdFiEECc3SW1JEo3Z49u6oDMUhUxl5kaUFAlz/ysgACgkQDMUhUxl5
kaVLhw/+Lssvjn2oDFR4hQeKfdMduIeytGf+LgPkqykxFXDcA2ujIoDf3gdTpBmz
SNJbgwh+ukigaaEHJ1FVfJUXnyQrgkhZJzF9Yvcx5BXMh2BjA/h33W6yEf2QJG7/
vWyQSZpRem2O8ynbZ6sxyeoUL3IpiMG7wdsJ53V5Z8pGN/ZVN3e93VOjJYLUwpuO
WPyrnbRwKAxRvuNdbQG8vks99U0JXXK8tD4+6fwfh4Po8B+BDyhMbTcClnsXyfVY
d7SEtAaARqKo/dmueWpBnx7BqmyAniYqVnAjbwh2u2S1ZnbOny0KwJM/61ro4p83
23Ci2VL7/QE2ew4LVFTs9OZezHhDe9cVP0xML/0BdXriknZYmdPXcLVO9NyMeWQJ
vCjjVu3hMefqWeCF6qsPrZ6ZyE8buoVkUAppxRvHrcemLorHxREFbPo85diHyd7c
w7KnjVZ4LjcsVGahWHeyk6ufOwtv/bQ828x9x8WMPZ/Jcn92hNafjgHWoYJAxXi5
wzKLFDiZO3qqlAKOzF74ALf3jGWmvKEMUgNkdMuVPS9bNPvVeoOYXWDIZ3XGc++o
mbwPWcuDhAW1ioVE5UMLzZWVYbC+06Ec6FEEux2fpdPv0atLefGyc+YLJ5E/zQUC
7KgG52HCwg2hDKSqZscbG2pbKoDNLI4R2sh91CZfKZeprO5xjDQ=
=ebkx
-----END PGP SIGNATURE-----
```

**Comment by laanwj** [comment]
Thanks! POsthumous utACK 993aa414d3c54f760c8ae4eb39cb49363acc7f33
